stripe/api_resources/abstract/test_helpers.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import, division, print_function
+
+from stripe import error, util, six
+from stripe.six.moves.urllib.parse import quote_plus
+
+
+class APIResourceTestHelpers:
+    def __init__(self, resource):
+        self.resource = resource
+
+    @classmethod
+    def class_url(cls):
+        if cls == APIResourceTestHelpers:
+            raise NotImplementedError(
+                "APIResourceTestHelpers is an abstract class.  You should perform "
+                "actions on its subclasses (e.g. Charge, Customer)"
+            )
+        # Namespaces are separated in object names with periods (.) and in URLs
+        # with forward slashes (/), so replace the former with the latter.
+        base = cls._resource_cls.OBJECT_NAME.replace(".", "/")
+        return "/v1/test_helpers/%ss" % (base,)
```

Comment:
Should there be a second `s` here (`%ss`)?

Reply:
Yep, the `OBJECT_NAME` is singular: 

```
    OBJECT_NAME = "terminal.reader"
```

Reply:
Oh I see this is just the same implementation as APIResource. Thanks for confirming!